tests/Components/test_BaseComponent.py

```diff
@@ -23,3 +53,11 @@ def test_append_child(base_component):
     base_component.append_child(child_component)
     assert(base_component.children["wing1"] == child_component)
     assert(base_component.wing1 == child_component)
+
+
+def test_angle_of_attack(freebody_component, attached_component):
+    npt.testing.assert_almost_equal(
+        freebody_component.angle_of_attack, -0.785, decimal=3)
```

Comment:
As the angle was set in degrees, it makes sense to test it in degrees, so the developer maintaining it does not need to make the mental conversion all the time.